content/docs/next-release-v2/architecture.md

```diff
@@ -43,21 +42,21 @@ run in a single process, or as a scalable distributed system. There are two main
 
 In this deployment Jaeger receives the data from traced applications and writes it directly to storage. The storage must be able to handle both average and peak traffic. Collectors use an in-memory queue to smooth short-term traffic peaks, but a sustained traffic spike may result in dropped data if the storage is not able to keep up.
 
-![Architecture](/img/architecture-v1-2023.png)
+![Architecture](/img/architecture-v2-2024.png)
 
 ### Via Kafka
 
 To prevent data loss between collectors and storage, Kafka can be used as an intermediary, persistent queue. Jaeger can be deployed with OpenTelemetry to handle writing the data to Kafka and pulling it off the queue and writing the data to the storage. Multiple Jaeger instances can be deployed to scale up ingestion; they will automatically partition the load across them.
 
-![Architecture](/img/architecture-v2-2023.png)
+![Architecture](/img/architecture-v2-kafka-2024.png)
```

Comment:
we should drop "and indexer" from the diagram, we never talk about "indexer" (this was a custom functionality in Uber's installation)

Reply:
Had the wrong images, now it should be fixed on this push.